### DISCUSSION TITLE
How to fix Unresolved requirement to other modules at runtime

### DISCUSSION THREAD

**Discussion by bllngr** [JaneWardSandy/eclipse-pde-partial-idea#121] [Q&A]
I've succesfully setup my project with multiple modules to build, but I can't get it to run successfully due to unresolved requirements between the main module and its dependencencies:

![1](https://github.com/JaneWardSandy/eclipse-pde-partial-idea/assets/303458/6487176a-f480-4552-ae28-fc7891dd7942)

I've tried `Resolve Manifest` and `Resolve Workspace` already, and manually adding the modules as Runtime dependencies in the project settings. What am I missing?

**Comment by JaneWardSandy** [comment]
@bllngr Have you added `PDE Facet` to `Geoplaner.batik`? Add facet for `Geoplaner.batik`, then `Resolve Workspace`, it will add module dependency to `Geoplaner` automatic.

<img width="818" alt="image" src="https://github.com/JaneWardSandy/eclipse-pde-partial-idea/assets/15345355/a99306a0-d6f6-42fc-99b4-55fded4f5fcc">
<img width="1452" alt="image" src="https://github.com/JaneWardSandy/eclipse-pde-partial-idea/assets/15345355/8ef06be3-db27-4557-bf92-577ef4a35b39">

Demo project for fig 
[pde.multi.zip](https://github.com/JaneWardSandy/eclipse-pde-partial-idea/files/14631597/pde.multi.zip)
